docs_user/modules/proc_migrating-tls-everywhere.adoc

```diff
@@ -117,18 +115,17 @@ EOF
 $IPA_SSH rm /tmp/freeipa.p12
 ----
 
+.Verification
 
-. Verify that the necessary resources were created by using the following commands:
+* Verify that the necessary resources are created:
 +
 ----
-oc get issuers -n openstack
+$ oc get issuers -n openstack
 ----
 +
 ----
-oc get secret rootca-internal -n openstack -o yaml
+$ oc get secret rootca-internal -n openstack -o yaml
 ----
 
-
 [NOTE]
-After the adoption procedure is finished, the cert-manager operator is responsible for issuing and refreshing new certificates when they expire.
-However, only the usual certificate rotation procedure is applied. This means that services and workloads that are not restarted and don't support reloading of the certificates are still using the certificates originally issued by IPA and generated by the certmonger daemon. In particular, as of writing this procedure, TLS certs for VNC consoles of the libvirt managed qemu domains are not rotated. You may need to restart or migrate the domains before the certificates expire.
+After the adoption procedure is finished, the cert-manager operator issues and refreshes new certificates when they expire. However, only the usual certificate rotation procedure is applied. This means that services and workloads that are not restarted and do not support reloading the certificates are still using the certificates originally issued by IPA and generated by the certmonger daemon. Particularly, TLS certs for VNC consoles of the libvirt-managed qemu domains are not rotated. You might need to restart or migrate the domains before the certificates expire.
```

Comment:
What is the “usual certificate rotation” procedure?

Reply:
Yeah, this isn't well worded, but the basic idea is as follows:

When certs expire, the cert-manager operator issues new certs and updates the cert-secrets with the new certs.  On the control plane, this will trigger a reconciliation and the pods on the control plane will automatically be restarted to pick up the new certs.  On the dataplane, a new deployment will need to be initiated manually - which will need to restart certain processes - to pick up the new certs.  Until then, the old certs will remain active.

Reply:
@vakwetu What do you think of the following rewrite? 

"The cert-manager operator issues new certificates and updates the secrets with the new certificates. As a result, the pods on the control plane automatically restart in order to obtain the new certificates. On the data plane, you must manually initiate a new deployment and restart certain processes to use the new certificates. The old certificates remain active until both the control plane and data plane obtain the new certificates."

One follow-up question: Can we give an example of the "certain processes" that must be restarted on the data plane?

Reply:
Hi @vakwetu, do you mind checking my proposed rewrite of the note?

"The cert-manager operator issues new certificates and updates the secrets with the new certificates. As a result, the pods on the control plane automatically restart in order to obtain the new certificates. On the data plane, you must manually initiate a new deployment and restart certain processes to use the new certificates. The old certificates remain active until both the control plane and data plane obtain the new certificates."

One follow-up question: Can we give an example of the "certain processes" that must be restarted on the data plane?